packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -127,7 +127,7 @@ const metadata: Meta<TableArgs> = {
             id-field-name="${x => dataSetIdFieldNames[x.data]}"
             data-unused="${x => x.updateData(x)}"
         >
-            <nimble-table-column-text field-name="firstName" placeholder="no value" column-id="first-name-column" action-menu-slot="name-menu" action-menu-label="Configure name">
+            <nimble-table-column-text sort-direction="ascending" sort-index="0" field-name="firstName" placeholder="no value" column-id="first-name-column" action-menu-slot="name-menu" action-menu-label="Configure name">
```

Comment:
We should document these APIs as part of #1044. No action needed right now, just mentioning to establish a link from that issue to this PR to help remind us.

Reply:
I've expanded the description of that issue to also include the new API added in this PR.